include/aikido/perception/DetectedObject.hpp

```diff
@@ -0,0 +1,91 @@
+#ifndef AIKIDO_PERCEPTION_DETECTEDOBJECT_HPP_
+#define AIKIDO_PERCEPTION_DETECTEDOBJECT_HPP_
+
+#include <string>
+#include "aikido/io/yaml.hpp"
+
+namespace aikido {
+namespace perception {
+
+/// DetectedObject delegates a detected object from a third-party
+/// perception algorithm.
+
+/// A perception algorithm should send information for an object via ROS
+/// visualization_msgs/Marker message like the following:
+/// Marker.ns + "_" + Marker.id -> dartUid (identity in DART world)
```

Comment:
I'm a bit confused about calling it `dartUid` because DART does not assign uid to an object afaik. But I think it makes keeps the skeleton names to be unique. Does `dartUid` refer to the name of the DART object, i.e. `dartSkeleton->getName() == dartUid`? Also is it DART world or Aikido world?